flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why so?

Reply:
I do not own an ARM device for testing. Other than that, i see no reason to restrict building to x86